lighthouse-cli/bin.ts

```diff
@@ -0,0 +1,251 @@
+/**
```

Comment:
to retain the bin symlinks that people already have setup. 

index.js is still present we just pulled all the logic into bin here.